src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -673,26 +708,53 @@ private void buildDockerImage() throws PluginExecutionException {
 
     private void startContainer() {
         try {
-            containerID = execDockerCmd(getContainerCommand(), 30);
-            info("docker container: " + containerID);
-        } catch (RuntimeException r) {
-            error("Error starting container: " + r.getMessage());
-            throw r;
+            String startContainerCommand = getContainerCommand();
+            debug("startContainer, cmd="+startContainerCommand);
+
+            ProcessBuilder processBuilder = new ProcessBuilder();
+            processBuilder.command(getCommandTokens(startContainerCommand));
+            processBuilder.redirectOutput(Redirect.INHERIT);
+            dockerRunProcess = processBuilder.start();
+            dockerRunProcess.waitFor();
```

Comment:
Is the expected use case that if the container fails to start the user hits ctrl-c and investigates? Otherwise I would have expected a 5 minute timeout and an error message if the container didn't start.

Reply:
If the container fails to start, the `waitFor()` should return and it should enter the following `if (dockerRunProcess.exitValue() != 0` section.  

Or if you mean when the server takes a long time to start, serverStart()'s `String startMessage = serverTask.waitForStringInLog(START_SERVER_MESSAGE_PREFIX, serverStartTimeoutMillis, messagesLogFile);` section should check for the message in messages.log and timeout.

Reply:
Ok, I thought we were on the dev mode "UI" thread at this point and we were going to hang. This looks good.